src/tests/ftest/util/command_utils.py

```diff
@@ -149,17 +159,22 @@ def _run_process(self):
         # Clear any previous run results
         self.result = None
         command = str(self)
+        if raise_exception is not None:
+            do_raise = raise_exception
+        else:
+            do_raise = self.exit_status_exception
+
         try:
             # Block until the command is complete or times out
             self.result = run_command(
-                command, self.timeout, self.verbose, self.exit_status_exception,
+                command, self.timeout, self.verbose, do_raise,
                 self.output_check, env=self.env)
 
         except DaosTestError as error:
             # Command failed or possibly timed out
             raise CommandFailure(str(error))    # pylint: disable=raise-missing-from
 
-        if self.exit_status_exception and not self.check_results():
+        if do_raise and not self.check_results():
```

Comment:
And to follow on from the previous two comments:
```suggestion
        if raise_exception and not self.check_results():
```